app/database/migration/server/index.ts

```diff
@@ -31,4 +32,15 @@ export default schemaMigrations({migrations: [
             }),
         ],
     },
+    {
+        toVersion: 3,
+        steps: [
+            addColumns({
```

Comment:
This is more a note to myself (or to the platform team) but we should consider adding unit tests for migrations.